tidb/src/test/java/org/tikv/tibigdata/tidb/TestSqlUtils.java

```diff
@@ -1,4 +1,4 @@
-package org.tikv.bigdata.tidb;
+package org.tikv.tibigdata.tidb;
```

Comment:
Looks like there are tests not using org.tikv.bigdata as package name

Reply:
fixed